frontend/src/components/ProjectListSidePanel.vue

```diff
@@ -52,6 +55,22 @@ export default {
         });
     });
 
+    const kbarActions = computed((): Action[] => {
+      const actions = outlineItemList.value.map((proj: any) =>
+        createAction({
+          id: `projects-${proj.id}`,
```

Comment:
I suggest to employ a namespace pattern and use the reversed domain name format like bb.project.${proj.id}.

1. Consistent with other places.

2. Friendly for future extension, especially for third party to register their actions.

Reply:
agree

Reply:
re-organized `action.id`, using dot (".") separated namespaces.
see [b46209f2](https://github.com/bytebase/bytebase/pull/100/commits/b46209f2591afa709961133ea769306f45d123e4)